arex-web-core/src/main/java/com/arextest/web/core/business/util/SchemaUtils.java

```diff
@@ -0,0 +1,135 @@
+package com.arextest.web.core.business.util;
+
+import com.arextest.web.common.LogUtils;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+public class SchemaUtils {
+    private static final String DEFAULT_STR = "String";
+    private static final int DEFAULT_INT = 1;
+    private static final double DEFAULT_DOUBLE = 1.0;
+    private static final char DEFAULT_CHAR = 'c';
+    private static final String NULL_STR = "NULL";
+
+    public static void mergeMap(Map<String, Object> contract, Map<String, Object> model) {
+        model.forEach((key, value) -> mergeEntry(contract, key, value));
+    }
+
+    public static String mergeJson(String contract, String model) {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            Map<String, Object> contractMap = contract == null ? new HashMap<>() : objectMapper.readValue(contract, Map.class);
+            if (contractMap == null) {
+                contractMap = new HashMap<>();
+            }
+            if (model == null) {
+                return contract;
+            }
+            Map<String, Object> modelMap = objectMapper.readValue(model, Map.class);
+            mergeMap(contractMap, modelMap);
+            return objectMapper.writeValueAsString(contractMap);
+        } catch (JsonProcessingException e) {
+            LogUtils.error(LOGGER, "objectMapper readValue failed! contract:{}, model:{}", contract, model);
+        }
+        return null;
+    }
+
+    private static void mergeEntry(Map<String, Object> contract, String key, Object value) {
+        if (value == null) {
```

Comment:
v in the new contract is null, won't this k-v exist in the aggregation contract?